packages/fluent/scss/chip/_layout.scss

```diff
@@ -0,0 +1,198 @@
+@use "_variables.scss" as *;
+@use "../core/_variables.scss" as *;
+
+@mixin kendo-chip--layout() {
+
+    // Chip
+    .k-chip {
+        padding-inline: var( --INTERNAL-kendo-chip-padding-x, #{ $kendo-chip-padding-x} );
+        padding-block: var( --INTERNAL-kendo-chip-padding-y, #{ $kendo-chip-padding-y } );
+        border-width: var( --kendo-chip-border-width, #{ $kendo-chip-border-width } );
+        border-style: solid;
+        outline: 0;
+        font-family: var( --kendo-font-family, #{ $kendo-font-family } );
+        font-size: var( --INTERNAL-kendo-chip-font-size, #{ $kendo-chip-font-size } );
+        line-height: var( --INTERNAL-kendo-chip-line-height, #{ $kendo-chip-line-height } );
+        display: inline-flex;
+        flex-flow: row nowrap;
+        align-items: center;
+        justify-content: center;
+        gap: var( --kendo-chip-spacing, #{ $kendo-chip-spacing } );
+        position: relative;
+        overflow: hidden;
+        cursor: pointer;
+        user-select: none;
+
+        *,
+        *::before,
+        *::after {
+            box-sizing: border-box;
+        }
+
+        .k-selected-icon-wrapper {
+            display: none !important; // sass-lint:disable-line no-important
+        }
+
+        &:hover,
+        &:focus {
+            outline: 0;
+            text-decoration: none;
+        }
+
+        &.k-chip-outline.k-focus::after,
+        &.k-chip-outline:focus::after {
+                content: '';
+                position: absolute;
+                border: medium none;
+                inset: var( --kendo-chip-focus-offset, #{ $kendo-chip-focus-offset } );
+                outline-width: var( --kendo-chip-focus-outline-width, #{ $kendo-chip-focus-outline-width } );
+                outline-style: var( --kendo-chip-focus-outline-style, #{ $kendo-chip-focus-outline-style } );
+                z-index: 1;
+        }
+    }
+
+
+    // Chip content
+    .k-chip-content {
+        padding: .5em 0;
+        margin: -.5em 0;
+        min-width: 0;
+        display: flex;
+        flex-flow: row nowrap;
+        align-items: center;
+        overflow: hidden;
+        flex: 1 1 auto;
+    }
+    .k-chip-content:first-child {
```

Comment:
The margin is needed because the content (text only) will be squashed and will not have any room to breath - a.k.a will be stuck on the left / right border of the chip.

Reply:
There is padding on `k-chip-{SIZE}` that prevents the text from being stuck on the left/right border.
Also, the spacing is currently inconsistent - there is more spacing on the left (see the screenshot below):

![content](https://user-images.githubusercontent.com/7753940/187672250-0773b22e-7b6b-45dd-991b-4c5251d4fd07.png)

I simply do not understand why we need this margin.
The `k-chip` element is `display: flex` and we have a `gap` on it which takes care of the spacing between the elements in the chip (icon/avatar, text, close icon).

If we need more spacing on the sides of the chip then we should increase the padding on `k-chip-{SIZE}` instead of using this "strange" margin.

I also do not understand why we need this padding and then a negative margin on `k-chip-content` but this is a different story:

![negative](https://user-images.githubusercontent.com/7753940/187673484-ff20249c-631c-49e3-831c-2498e2b732ce.png)

@Juveniel @TeyaVes @epetrow This affects all themes so your input will be appreciated :)

Reply:
This is needed because this balances the spacing between the left border and the start of the text / the icon and the right border. It comes from a fix.